examples/data-objects/webflow/package.json

```diff
@@ -12,15 +12,15 @@
 	"license": "MIT",
 	"author": "Microsoft and contributors",
 	"sideEffects": false,
+	"type": "module",
 	"main": "dist/index.js",
-	"module": "lib/index.js",
+	"module": "dist/index.js",
```

Comment:
do we care more that `dist` remains for CJS and `lib` is for `ESNext` modules? or that `dist` is a package's 'default built file location'?

i'm indifferent. it's not hard to change at any point either, so dunno if it's particularly important we get it right immediately.

Reply:
Hmm, it does seem like we need to decide on a convention (lib vs. dist). Ideally we could remove the CommonJS build altogether and just make a uniform decision. I don't feel strongly about lib vs. dist but the in-between state might be a bit confusing since dist has traditionally been the CJS build. I wonder if using lib here makes more sense?

Reply:
Yeah, after publishing this PR I've also leaned toward keeping "dist = cjs", "lib = esnext" as simpler. I'll push an update.

Reply:
looks like that policy is consistent with some updates Craig did to our other web-only examples, see e.g. #10845. cool. Updated.

Reply:
Tony brought up a good point on the e2e test PR--our launch target configuration for debugging mocha tests takes a dependency on the build output being in "dist". I'm going to revert the above change and add a note to the tsconfig calling that out. The examples I pointed out above don't have this problem as they use jest for tests.